src/edge/tests/test_recombine_annotations.py

```diff
@@ -285,3 +286,189 @@ def test_adds_annotations_correctly_with_reverse_cassette(self):
         self.assertEquals(a.feature.strand, 1)
         self.assertEquals(a.base_first, len(self.upstream) + len(self.front_bs) + 1)
         self.assertEquals(a.base_last, len(self.upstream) + len(self.front_bs) + len(self.middle))
+
+    def test_preserves_annotations_on_homology_arm_fwd_that_have_not_changed(self):
+        cassette = ''.join([self.front_bs, self.back_bs])
+
+        # add annotaiton on upstream arm
+        self.fragment.annotate(len(self.upstream) + 2,
+                               len(self.upstream) + len(self.front_bs),
+                               "Up arm", "feature", 1)
```

Comment:
what does 1 mean here? I know it's strand but what does strand of 1 mean? also what does strand -1 mean? https://github.com/ginkgobioworks/edge/blob/110d0d2778a050922ce97477e382da0bd1284625/src/edge/models/fragment_annotator.py#L11

Reply:
Fwd = 1, rev = -1